core/controllers/acl_decorators.py

```diff
@@ -1778,7 +1736,7 @@ def test_can_edit_story(self, topic_id, **kwargs):
         if not self.user_id:
             raise base.UserFacingExceptions.NotLoggedInException
 
-        topic_rights = topic_services.get_topic_rights(topic_id)
+        topic_rights = topic_services.get_topic_rights(topic_id, strict=False)
```

Comment:
Do we even use `strict=True` anywhere? Maybe we should be getting rid of that functionality?

Reply:
Yes we do. For example here(default is strict=True): https://github.com/oppia/oppia/blob/95dad9010faf529a0309cb7d629b84943ccb4731/core/domain/topic_services.py#L1006

Reply:
Thanks for pointer, SGTM.